deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -422,6 +422,44 @@ topologyUpdater:
   annotations: {}
   affinity: {}
 
+topologyGC:
+  enable: false
```

Comment:
In the new setup we want to enable topology-gc by default

```suggestion
  enable: true
```

Reply:
Is it make sense to enable `topologyGC` by default when `topologyUpdater` is disabled by default(as well as creating NRT CRDs)?

Reply:
My bad :) forgot that we only deploy `GC` when `updater` is enabled